bittensor/extrinsics/unstaking.py

```diff
@@ -110,7 +110,7 @@ async def unstake_extrinsic(
         prompt (bool): If ``true``, the call waits for confirmation from the user before proceeding.
 
     Returns:
-        success (bool): Flag is ``true`` if extrinsic was finalized or included in the block. If we did not wait for finalization / inclusion, the response is ``true``.
+        success (bool): Flag is ``true`` if extrinsic was finalized or uncluded in the block. If we did not wait for finalization / inclusion, the response is ``true``.
```

Comment:
What is 'uncluded'?

Reply:
typo for some reason. there isn't any other explanation.